jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/SlideGroupsTests.java

```diff
@@ -31,109 +27,74 @@ public void before() {
 
     @Test
     public void activeClassSlideGroupTests() {
+        activeClassSlideGroup.startTest();
         activeClassSlideGroup.is().displayed();
-        testCommonSlideGroup(activeClassSlideGroup);
+
+        activeClassSlideGroup.clickOnSlideByIndex(1);
+        activeClassSlideGroup.has().slideSelected(1);
+        activeClassSlideGroup.clickOnSlideByIndex(2);
+        activeClassSlideGroup.has().slideSelected(2);
+        activeClassSlideGroup.has().slideNotSelected(1);
+
+        if (activeClassSlideGroup.nextButtonIsActive()) {
+            activeClassSlideGroup.clickOnNextButton();
+            activeClassSlideGroup.clickOnSlideByIndex(2);
+            activeClassSlideGroup.has().slideSelected(2);
+
+            activeClassSlideGroup.clickOnSlideByIndex(3);
+            activeClassSlideGroup.has().slideSelected(3);
+            activeClassSlideGroup.has().slideNotSelected(2);
+        }
     }
 
     @Test
     public void centerActiveSlideGroupTests() {
-        List<Integer> slidesPositions = new SlideGroupTestsData().centerActiveSlideGroupTestData();
+        centerActiveSlideGroup.startTest();
         centerActiveSlideGroup.is().displayed();
-        for (int i = 1; i <= 3; i++) {
-            centerActiveSlideGroup.clickOnSlideByIndex(i);
-            centerActiveSlideGroup.has().slideSelected(i);
-        }
-        for (Integer slidesPosition : slidesPositions) {
-            centerActiveSlideGroup.clickOnSlideByIndex(4);
-            centerActiveSlideGroup.has().visibleSlidesPosition(slidesPosition);
-            centerActiveSlideGroup.has().slideSelected(3);
-        }
-        for (int i = 4; i <= 6; i++) {
-            centerActiveSlideGroup.clickOnSlideByIndex(i);
-            centerActiveSlideGroup.has().slideSelected(i);
-            centerActiveSlideGroup.has().visibleSlidesPosition(-1316);
-        }
-    }
+        centerActiveSlideGroup.clickOnSlideByIndex(2);
+        centerActiveSlideGroup.has().centerActiveSlide();
 
-    @Test
-    public void customIconsSlideGroupTests() {
-        customIconsSlideGroup.is().displayed();
```

Comment:
check coverage of customIconsSlideGroup. Currently it's not tested into the branch. Branch [vuetify-develop](https://github.com/jdi-testing/jdi-light/tree/vuetify-develop) has such check now.